docs/source/python/csv.rst

```diff
@@ -75,7 +75,22 @@ Customized conversion
 ---------------------
 
 To alter how CSV data is converted to Arrow types and data, you should create
-a :class:`ConvertOptions` instance and pass it to :func:`read_csv`.
+a :class:`ConvertOptions` instance and pass it to :func:`read_csv`::
+   import pyarrow as pa
```

Comment:
Can you leave a blank line between the code block and the paragraph above? (restructuredtext formatting is quite strict about those details ..)

Reply:
Before and after screenshots of the rendering, if it helps with review.

<img width="774" alt="Screen Shot 2021-02-19 at 5 18 21 PM" src="https://user-images.githubusercontent.com/878798/108575784-9c8e3580-72d8-11eb-91b7-426b9b570261.png">

<img width="745" alt="Screen Shot 2021-02-19 at 5 19 31 PM" src="https://user-images.githubusercontent.com/878798/108575787-9ef08f80-72d8-11eb-927d-21cc07cf32e1.png">

Reply:
And here's me testing the example executes without exception:

```
>>> from pyarrow import csv
>>> import pyarrow as pa
>>> table = csv.read_csv(
...     "tips.csv",
...     convert_options=pa.csv.ConvertOptions(
...         column_types={
...             "total_bill": pa.decimal128(precision=10, scale=2),
...             "tip": pa.decimal128(precision=10, scale=2),
...         }
...     ),
... )

```


Note: It took me a while to find the dataset in question. Looks like it's in the pandas repo. A link to it from these docs would be nice.

https://raw.githubusercontent.com/pandas-dev/pandas/master/pandas/tests/io/data/csv/tips.csv